File: app/src/main/java/id/assel/caribengkel/model/Workshop.java
package id.assel.caribengkel.model;

import com.google.firebase.firestore.GeoPoint;

public class Workshop {
    public int id;
    public GeoPoint latLng;
    public String name;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public GeoPoint getLatLng() {
        return latLng;
    }

    public void setLatLng(GeoPoint latLng) {
        this.latLng = latLng;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: app/src/main/java/id/assel/caribengkel/model/Workshop.kt
package id.assel.caribengkel.model

import com.google.firebase.firestore.GeoPoint

data class Workshop constructor(
    var id: Int,
    var latLng: GeoPoint,
    var name: String,
    var active: Boolean,
    var profile: Profile? = null
) {
    var currentOrderUuid: String? = null


    //empty constructor for firestore
    constructor(): this(0, GeoPoint(0.0,0.0), "", false, Profile())

    override fun toString(): String {
        return name
    }
}
